catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/proposals/proposals_cubit.dart

```diff
@@ -14,11 +16,27 @@ final class ProposalsCubit extends Cubit<ProposalsState> {
 
   /// Loads the proposals.
   Future<void> load() async {
-    final proposals = await proposalRepository.getDraftProposals();
+    const currentCampaignId = 'f14';
+
+    final proposals = await proposalRepository.getDraftProposals(
+      campaignId: currentCampaignId,
+    );
+
+    final pendingProposals = proposals.map(
+      (proposal) {
+        return PendingProposal.fromProposal(
+          proposal,
+          campaignName: 'F14',
+          formattedFundsRequested: CryptocurrencyFormatter.formatAmount(
+            proposal.fundsRequested,
```

Comment:
Cubit shouldn't be formatting string because often you need build context for it which you cannot use in cubit.

Would it make sense to format inside `PendingProposal.fromProposal`?

Reply:
Agree, moved to getter inside `PendingProposal`. Later it may be a function which accepts `BuildContext`